core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -629,6 +639,8 @@ public void run() {
 
     private static final int MUTATION_BATCH_SIZE = 1 << 17;
     private final ExecutorService sendThreadPool;
+    private final LinkedTransferQueue<Runnable> queue = new LinkedTransferQueue<>();
+    private final ThreadPoolExecutor executor = new ThreadPoolExecutor(1, 1, 10L, TimeUnit.SECONDS, queue);
```

Comment:
Would be nice to name thread.  Also it should be a daemon thread (I think it will not be by default).  These are things I think that SimpleThreadPool does (may have name wrong, you were using it earlier).